core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -1,4 +1,4 @@
-<div class="audio-header">
+<div class="audio-header" ng-if="isAudioLanguageCodeAvailable()">
```

Comment:
Hm, shouldn't the condition be on whether there's anything to populate the dropdown with, instead?

Reply:
Yes that seems to be more appropriate. Will change it

Reply:
Changed it to the array length of the languagesInExploration array which is used in the  dropdown.